server/controllers/authController.js

```diff
@@ -0,0 +1,73 @@
+import bcrypt from 'bcrypt';
+import authHelper from '../helpers/authHelpers';
+
+/**
+ * This class contains all methods
+ * required to handle all
+ * authentication routes.
+ */
+class authController {
+  /**
+   * This method handle the signup request.
+   * @param {object} req The user's request.
+   * @param {object} res The response.
+   * @returns {object} The status and some data of the user.
+   */
+  static async signup(req, res) {
+    const { lastname, firstname, email } = req.body;
+    const password = await bcrypt.hash(req.body.password, 10);
```

Comment:
destructuring was finished

Reply:
Actually I think , for the matter of security the password shouldn't be copied to any variable instead it should be hashed directly from the body. That's why I didn't destruct it from the body.

Reply:
The security matters when the password is stored in the database without being hashed, but as long as it will be stored hashed, It is okay. The password is still secure and in case of any security breach, the cybercriminals won't get access to the passwords.

Reply:
Ok already done that!!